exercises/mongo-shell/basic-mongo-queries.md

```diff
@@ -1,7 +1,157 @@
-## READ Operations
+# CRUD Operations
+
+This document deals with basic create, read, update and delete operations using the mongo shell. 
+
+[Read more](https://docs.mongodb.com/manual/crud/) :book: about the CRUD operations here. 
+
+## Create Operations
+
+A MongoDB database is made up of 1 or more collections which are in turn made up of 1 or more documents. A document has a `key: value` structure, similar to JSON objects. In the following exercises, we'll learn how to insert documents using the following operations:
+* `insertOne()`
+* `insertMany()` 
+
+**Exercise 1** :computer: 
+
+* Verify what database you are currently using (should be `test`, this is the default.)
+
+    ```javascript
+    db
+    ```
+
+* Let's create a new database called `employee`. 
+    ```javascript
+    use employee_db
+    ```
+
+:arrow_right: Note that `use` creates a new database, if one does not already exist. 
+
+* Next, let's create a collection called `employee_info`.
+
+    ```javascript
+    db.createCollection("employee_info")
+    ```
+    
+* Let's confirm that the collection was successfully created.
+
+    ```javascript
+    show collections
+    ```
+:arrow_right: Similarly, you can also use `show dbs` to obtain a list of the all the databases. 
+
+* Let's try to insert the following `key:value` pairs as a document in our collection. 
+
+    {
+        "fname": "john",
+        "lname": "doe",
+        "salary": 70000,
+        "departments": ["sales", "admin"],
+        "hiredate": "2018-08-29"
+      }
+
+    **query:**
+    ```javascript
+    db.employee_info.insertOne({
+        "fname": "john", 
+        "lname": "doe", 
+        "salary": 70000, 
+        "departments": ["sales", "admin"], 
+        "hiredate": "2018-08-29"})
+    ```
+
+    **result:**
+    ```javascript
+    {
+        "acknowledged" : true,
+        "insertedId" : ObjectId("5d8e6f9ccaa4f8ddbe27296f")
+    }
+    ```
+    The response to this query is a document in itself as seen below. `"acknowldeged": true` means we the insert was successful. Also note the Object ID which is the unique identifier for this document. Every document has its own unique `_id`. It is not necessary to specify `_id` in the query in which case, MongoDB will automatically add one to the document. 
+
```

Comment:
```suggestion
➡️ Letting Mongo take care of your `_id` field is a good idea.  Mongo will never try to assign a duplicate value to this field, sometimes we make mistakes and may try to do that.  Mongo is pretty relaxed and sometimes doesn't stop us from doing reckless things such as assigning the same `_id` to more than one Object.  If you need control over an `id` field for you document consider creating one instead of overwriting the object id.```